docker/DockerVerify.sh

```diff
@@ -67,11 +67,11 @@ TEST_DIRECTORY="${docker_dir}/test/integration"
 export TEST_DIRECTORY
 
 # Add --no-logcapture to see logs interleaved with the test output
-BEHAVE_OPTS=(-f pretty --logging-level INFO --logging-clear-handlers --tags ~@no-ci)
```

Comment:
I moved the filtering to https://github.com/apache/nifi-minifi-cpp/pull/1369/files#diff-aebf957cd74e48986733a1597cb988264e869aa6709d4c6990e7f3e564e4e384R75

Reply:
My point is that there would be no harm in keeping the tag, even if unused. The added complexity is minimal, and there may be cases in the future where we need to disable individual (e.g. long running) tests, not the whole extension. It can also be readded, so it's fine if you prefer to still remove it.

Reply:
The problem with the tag was that it influenced local runs as well. https://github.com/apache/nifi-minifi-cpp/pull/1349#discussion_r907500456
Due to this reason Id rather readd it later if we need it.